homeassistant/components/gogogate2/cover.py

```diff
@@ -107,15 +110,32 @@ def supported_features(self):
         """Flag supported features."""
         return SUPPORT_OPEN | SUPPORT_CLOSE
 
+    @property
+    def is_closing(self):
+        """Return if the cover is closing or not."""
+        return self._get_door_status() == TransitionDoorStatus.CLOSING
+
+    @property
+    def is_opening(self):
+        """Return if the cover is opening or not."""
+        return self._get_door_status() == TransitionDoorStatus.OPENING
+
     async def async_open_cover(self, **kwargs):
         """Open the door."""
         await self._api.async_open_door(self._get_door().door_id)
+        await self.coordinator.async_refresh()
 
     async def async_close_cover(self, **kwargs):
         """Close the door."""
         await self._api.async_close_door(self._get_door().door_id)
+        await self.coordinator.async_refresh()
 
     @property
     def extra_state_attributes(self):
         """Return the state attributes."""
-        return {"door_id": self._get_door().door_id}
+        return {**super().state_attributes, "door_id": self._get_door().door_id}
```

Comment:
Why are we accessing `state_attributes`?

Reply:
Its been so long, I have no idea.

Reply:
This PR pre-dates `extra_state_attributes`, but even then???.  I'll remove it